RELEASE.md

```diff
@@ -0,0 +1,16 @@
+# Release process
+
+bump up version `VERSION` file
+
+## Release crates
+
+Find crates that has been changed last release
+```
+cargo ws changed 
```

Comment:
Is this short for https://crates.io/crates/cargo-workspaces#changed?

Reply:
yup

Reply:
it would be handy if there are other tools that can show diff between published and what is in the workspace